charts/egeria-presentation/templates/ingress.yaml

```diff
@@ -0,0 +1,61 @@
+{{- if .Values.ingress.enabled -}}
+{{- $fullName := include "egeria-presentation.fullname" . -}}
```

Comment:
Are you making of this generally? Looks like a good potential pattern to look at further for our other charts

Reply:
This is the default output of `helm init`